graphprops.pl

```diff
@@ -0,0 +1,20 @@
+
+module(cycle).
```

Comment:
Whats the difference between this and `cycle.pl`?

Reply:
graphprops got renamed, not sure why its still here